static/docs/command-reference/remote/add.md

```diff
@@ -250,6 +250,17 @@ $ dvc remote add myremote gs://bucket/path
 $ dvc remote add myremote ssh://user@example.com/path/to/dir
 ```
 
+> **NOTE!** DVC requires SSH and SFTP access to work properly. Please use `ssh`
```

Comment:
@jorgeorpinel we should probably change name of the section from Example to Details? and mention above in the main text that there are important remote specific detail below.

Reply:
How about calling it **Supported types**?

Reply:
And yes about the mention in the **Description**.

Reply:
Yep. sounds good. May be specify `Supported storage types`?

Reply:
Updated.